src/utils/options.js

```diff
@@ -3,47 +3,47 @@ const jsonDescription =
 
 const messageDescription = `Specifies a source for providing a message to the command. If a string is provided directly as an argument, this option will be ignored. The message must be provided via an argument or via this option. Sources must be one of \`file\` or \`stdin\`. In the case of \`file\`, a corresponding identifier must also be provided.
 
-	Note: if both secret passphrase and message are passed via stdin, the passphrase must be the first line.
+Note: if both secret passphrase and message are passed via stdin, the passphrase must be the first line.
 
-	Examples:
-	- --message file:/path/to/my/message.txt
-	- --message stdin
+Examples:
```

Comment:
Why was there a format change here?

Reply:
I think this is because I made changes to a temporary file and then copied the file back, I will revert these changes.